app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -4,22 +4,56 @@
 # Make sure to set KUBELET_VERSION to an image tag published here:
 # https://quay.io/repository/coreos/hyperkube?tab=tags Alternatively,
 # override $KUBELET_ACI to a custom location.
+#
 
 set -e
 
-if [ -z "${KUBELET_VERSION}" ]; then
-    echo "ERROR: must set KUBELET_VERSION"
+function require_ev_all() {
+    for rev in $@ ; do
+        if [[ -z ${!rev} ]]; then
+            echo ${rev} is not set
+            exit 1
+        fi
+    done
+}
+
+function require_ev_one() {
+    for rev in $@ ; do
+        if [[ ! -z ${!rev} ]]; then
+            return
+        fi
+    done
+    echo One of $@ must be set
     exit 1
+}
+
+if [[ ! -z ${KUBELET_VERSION} ]]; then
+    echo KUBELET_VERSION environment variale is deprecated, please use KUBELET_IMG_TAG instead
 fi
 
-KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+KUBELET_IMG_TAG="${KUBELET_IMG_TAG:-${KUBELET_VERSION}}"
+KUBELET_IMG="${KUBELET_ACI}"
```

Comment:
That's right, it deprecates `KUBELET_ACI` which is kept only for backwards compatibility. I'll add a deprecation warning here too.